src/iop/scalepixels.c

```diff
@@ -191,8 +196,11 @@ void commit_params(dt_iop_module_t *self, const dt_iop_params_t *const params, d
                    dt_dev_pixelpipe_iop_t *piece)
 {
   dt_iop_scalepixels_params_t *p = self->params;
+  dt_iop_scalepixels_data_t *d = piece->data;
 
-  memcpy(piece->data, params, sizeof(dt_iop_scalepixels_data_t));
+  d->pixel_aspect_ratio = p->pixel_aspect_ratio;
+  d->x_scale = 1.0f;
+  d->y_scale = 1.0f;
 
```

Comment:
Oh, that.
Hm, i guess `params` passed as function argument should be used.
But i can not tell whether  ```dt_iop_scalepixels_params_t *p = self->params;``` is wrong.